packages/block-library/src/post-content/edit.js

```diff
@@ -153,11 +222,12 @@ export default function PostContentEdit( {
 		<RecursionProvider uniqueId={ contextPostId }>
 			{ contextPostId && contextPostType ? (
 				<Content
+					clientId={ clientId }
```

Comment:
Is this leftover from something else?

Reply:
No, it gets passed down to `<EditableContent />`, which needs it... for stuff 😄